core/storage/email/gae_models_test.py

```diff
@@ -128,9 +153,133 @@ def test_get_by_hash_behavior_with_sent_datetime_lower_bound(self):
+            created2 = email_models.GeneralFeedbackEmailReplyToIdModel.create(
+                'user_id', 'thread_id')
+            created2.put()
+
+    def test_get_by_reply_to_id_works_correctly(self):
+        # Name is too long for linter.
+        model = email_models.GeneralFeedbackEmailReplyToIdModel
+
+        created = model.create('user_id', 'thread_id')
+        created.put()
+
+        result = model.get_by_reply_to_id(created.reply_to_id)
+
+        self.assertNotEqual(result, None) #There should be a result
+        self.assertEqual(result.reply_to_id, created.reply_to_id)
+
+        result = model.get_by_reply_to_id('non_existent_reply_to_id')
+
+        self.assertEqual(result, None)
+
+    def test_get_works_correctly(self):
+        created = email_models.GeneralFeedbackEmailReplyToIdModel.create(
+            'user_id', 'thread_id')
+        created.put()
+
+        result = email_models.GeneralFeedbackEmailReplyToIdModel.get(
+            'user_id', 'thread_id', strict=False)
+
+        self.assertNotEqual(result, 1)
+
+        result = email_models.GeneralFeedbackEmailReplyToIdModel.get(
+            'bad_user_id', 'bad_thread_id', strict=False) #Should not throw
+        self.assertEqual(result, None)
+
+        with self.assertRaises(Exception):
+            result = email_models.GeneralFeedbackEmailReplyToIdModel.get(
+                'bad_user_id', 'bad_thread_id') #Should throw
+
+    def test_get_multi_by_user_ids_works_correctly(self):
+        # Name is too long for linter.
+        model = email_models.GeneralFeedbackEmailReplyToIdModel
+
+        model.create('user_id_1', 'thread_id')
+        model.create('user_id_2', 'thread_id')
+
+        result = model.get_multi_by_user_ids(
+            ['user_id_1', 'user_id_2'], 'thread_id')
+
+        self.assertEqual(len(result), 2)
+        assert 'user_id_1' in result.keys()
+        assert 'user_id_2' in result.keys()
+
+    def test_user_id(self):
+        result = email_models.GeneralFeedbackEmailReplyToIdModel.create(
+            'user_id', 'entity_type.entity_id.thread_id')
+
+        self.assertEqual(result.user_id, 'user_id')
+
+    def test_entity_type(self):
+        result = email_models.GeneralFeedbackEmailReplyToIdModel.create(
+            'user_id', 'entity_type.entity_id.thread_id')
+
+        self.assertEqual(result.entity_type, 'entity_type')
+
+    def test_entity_id(self):
+        result = email_models.GeneralFeedbackEmailReplyToIdModel.create(
+            'user_id', 'entity_type.entity_id.thread_id')
+
+        self.assertEqual(result.entity_id, 'entity_id')
+
+    def test_thread_id(self):
+        result = email_models.GeneralFeedbackEmailReplyToIdModel.create(
+            'user_id', 'entity_type.entity_id.thread_id')
+
+        self.assertEqual(result.thread_id, 'entity_type.entity_id.thread_id')
+
+
 class GenerateHashTests(test_utils.GenericTestBase):
+
     """Test that generating hash functionality works as expected."""
 
+
```

Comment:
one new line is sufficient, remove extra new line.

Reply:
And done, pushed

Reply:
This new line still seems present--was it removed?